src/renderer/Renderer.js

```diff
@@ -62,34 +62,6 @@ export default class Renderer {
         this._initGL(gl);
     }
 
-    /**
```

Comment:
this wasn't being used...? or is this somehow provided now by the layer?

Reply:
not being used, now it only uses the matrix